### PR TITLE
Make builds a bit faster, but mostly cleanup

### PR DESCRIPTION
**Pull request by welps**
This PR does the following:
- shaves off about 40% of the build time by doing map lookups instead
- remove extraneous code that was never used (social stuff)
- remove zod which was only used to do some validation that had no impact on the list output

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -29,82 +25,55 @@ export const TOKEN_LISTS: TokenListType = {
...
  */
-export const RawContractMapTokenSchema = z.object({
-  address: z.string(),
-  decimals: z.union([z.string(), z.number()]),
-  name: z.string(),
-  symbol: z.string(),
-});
+export interface RawContractMapTokenSchema {
+  address: string;
+  decimals: string | number;
```
Comment (brunobar79) [inline src/constants.ts:65]: I know this was here already, but why are we expecting decimals to be a string?

File: src/constants.ts [hunk 2]
```diff
@@ -29,82 +25,55 @@ export const TOKEN_LISTS: TokenListType = {
...
-  website: z.string().optional(),
-});
-
-export type RawContractMapToken = z.infer<typeof RawContractMapTokenSchema>;
-export type RawEthereumListsToken = z.infer<typeof RawEthereumListsTokenSchema>;
-export type Token = z.infer<typeof TokenSchema>;
-export type TokenSocialMetadata = z.infer<typeof SocialSchema>;
+export interface RawEthereumListsTokenSchema {
+  address?: string;
+  decimals?: string | number;
```
Comment (brunobar79) [inline src/constants.ts:75]: same

**Review by brunobar79 — APPROVED** [review]
💯

File: src/constants.ts (hunk 1 above)
Comment (welps) [reply src/constants.ts:65]: i don't know

File: src/constants.ts (hunk 2 above)
Comment (welps) [reply src/constants.ts:75]: leav eme alone

**Comment by welps** [comment]
RNBW-2860

**Pull request merged**